lib/types.js

```diff
@@ -446,7 +444,7 @@ Type.isType = function (/* any, [prefix] ... */) {
 };
 
 Type.__reset = function (size) {
-  debug('resetting type buffer to %d', size);
+  //debug('resetting type buffer to %d', size);
```

Comment:
I removed these because 'debug' was another node library that I didn't want to include in my bundle if I could avoid it. There were more bundle problems with it there so I removed it.